common/forms.py

```diff
@@ -47,3 +128,160 @@ class Meta:
             "password1",
             "password2",
         ]
+        exclude = [
+                "username",
+                ]
+
+    def clean_email(self) -> str:
+        """
+        Validate that the supplied email address is unique.
+
+        This runs after the basic `UserCreationForm` validation.
+        """
+        email = self.cleaned_data["email"].lower()
+        if User.objects.filter(email=email).exists():
+            self.add_error("email", ValidationError("Email already exists"))
+        return email
+
+    def add_error_classes(self):
+        """
+        Add bootstrap error classes to fields and move errors from password2 to password1
+        so that errors are displayed in one place on the left side of the form
+        """
+        super().add_error_classes()
+        if "password1" in self.errors or "password2" in self.errors:
+            self.fields["password1"].widget.attrs.update({'class': 'form-control is-invalid'})
+            self.fields["password2"].widget.attrs.update({'class': 'form-control is-invalid'})
+            errors_p1 = self.errors.get("password1", [])
+            self.errors["password1"] = errors_p1 + self.errors.get("password2", [])
+            if "password2" in self.errors:
+                del self.errors["password2"]
+
+
+class ProfileForm(BootstrapErrorFormMixin, forms.ModelForm):
+    affiliation = forms.ChoiceField(
+        widget=forms.Select(attrs={"class": "form-control", "placeholder": "University"}),
+        label="University affiliation",
+        choices=UNIVERSITIES,
+        label_suffix="",
+    )
+    department = forms.CharField(
+            widget= ListTextWidget(
+               data_list=DEPARTMENTS,
+               name='department-list',
+               attrs={"class": "form-control", "placeholder": "Department"}
+               ),
+            label="Select closest department name or enter your own",
+            label_suffix="",
+            required=False
+            )
+    why_account_needed = forms.CharField(
+            widget=forms.Textarea(
+                attrs={
+                    "class": "form-control",
+                    "placeholder": "Because you are not using Swedish University email, please describe why do you need "
+                    "an account.\nYour request will be submited for evaluation.*",
+                    "style": "height: 70px"
+                    }
+                ),
+            required=False,
+            )
+    note = forms.CharField(
+            widget=forms.Textarea(
+                attrs={
+                    "class": "form-control",
+                    "placeholder": ("If you would like us to get in touch with you, to answer your questions or provide "
+                    "help with Serve, please describe what do you need here"),
```

Comment:
I fixed it now, now it is a description of the field below rather than the placeholder.

>Also, it is a bit weird to have this during the registration step, right?
Indeed, an unusual approach. This is something we want to keep while the service is running in "beta" so that we can help users if needed. Better that they ask us than click around and leave without finding a solution. Will be removed at some point when we feel like it's not needed anymore.